File: mobile/src/main/java/treehou/se/habit/core/db/model/LinkedPageDB.java
package treehou.se.habit.core.db.model;

import io.realm.RealmObject;

public class LinkedPageDB extends RealmObject {

    private String id;
    private String link;
    private String title;
    private boolean leaf;
    //private RealmList<WidgetDB> widget;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public boolean isLeaf() {
        return leaf;
    }

    public void setLeaf(boolean leaf) {
        this.leaf = leaf;
    }

    /*public RealmList<WidgetDB> getWidget() {
        //return widget;
        return null;
    }

    public void setWidget(RealmList<WidgetDB> widget) {
        //this.widget = widget;
    }*/
}

File: mobile/src/main/java/treehou/se/habit/core/db/model/LinkedPageDB.kt
package treehou.se.habit.core.db.model

import io.realm.RealmObject

open class LinkedPageDB : RealmObject() {

    //private RealmList<WidgetDB> widget;

    var id: String? = null
    var link: String? = null
    var title: String? = null
    var leaf: Boolean = false

    /*public RealmList<WidgetDB> getWidget() {
        //return widget;
        return null;
    }

    public void setWidget(RealmList<WidgetDB> widget) {
        //this.widget = widget;
    }*/
}
